x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -31,6 +31,44 @@ MIN(languages):integer
 // end::min-result[]
 ;
 
+maxOfBoolean
+required_capability: agg_max_min_boolean_support
+from employees | stats s = max(still_hired);
```

Comment:
This test and `from employees | stats s = min(still_hired)` can be a single one, no need to run two queries that do `max` and `min` separately.

The same goes for the next pair of queries.